requirements copy.txt

```diff
@@ -0,0 +1,201 @@
+docker==6.1.3
+docker-pycreds==0.4.0
+docstring-parser==0.15
+einops==0.6.1
+exceptiongroup==1.1.2
+fancy-einsum==0.0.3
+fastapi==0.100.1
+filelock==3.12.2
+fonttools==4.42.0
+frozenlist==1.3.3
+fsspec==2023.6.0
+ghp-import==2.1.0
+gitdb==4.0.10
+GitPython==3.1.35
+google-auth==2.22.0
+google-auth-oauthlib==1.0.0
+griffe==0.36.2
+grpcio==1.56.2
+h11==0.14.0
+huggingface-hub==0.15.1
+hydra-core==1.3.2
+idna==3.4
+importlib-metadata==6.8.0
+importlib-resources==6.0.0
+inquirer==3.1.3
+itsdangerous==2.1.2
+jaxtyping==0.2.20
+Jinja2==3.1.2
+jmespath==1.0.1
+jsonargparse==4.23.0
+jsonschema==4.18.6
+jsonschema-specifications==2023.7.1
+kiwisolver==1.4.4
+lightning==2.0.6
+lightning-api-access==0.0.5
+lightning-cloud==0.5.37
+lightning-fabric==2.0.6
+lightning-utilities==0.9.0
+linkify-it-py==2.0.2
+lit==16.0.6
+Markdown==3.4.4
+markdown-it-py==3.0.0
+MarkupSafe==2.1.3
+matplotlib==3.7.2
+mdit-py-plugins==0.4.0
+mdurl==0.1.2
+mergedeep==1.3.4
+mkdocs==1.5.2
+mkdocs-autorefs==0.5.0
+mkdocs-gen-files==0.5.0
+mkdocs-literate-nav==0.6.1
+mkdocs-material==9.3.1
+mkdocs-material-extensions==1.1.1
+mkdocs-section-index==0.3.7
+mkdocstrings==0.23.0
+mkdocstrings-python==1.7.0
+mpmath==1.3.0
+multidict==6.0.4
+multiprocess==0.70.14
+networkx==3.1
+numpy==1.25.0
+nvidia-cublas-cu11==11.10.3.66
+nvidia-cuda-cupti-cu11==11.7.101
+nvidia-cuda-nvrtc-cu11==11.7.99
+nvidia-cuda-runtime-cu11==11.7.99
+nvidia-cudnn-cu11==8.5.0.96
+nvidia-cufft-cu11==10.9.0.58
+nvidia-curand-cu11==10.2.10.91
+nvidia-cusolver-cu11==11.4.0.1
+nvidia-cusparse-cu11==11.7.4.91
+nvidia-nccl-cu11==2.14.3
+nvidia-nvtx-cu11==11.7.91
+oauthlib==3.2.2
+omegaconf==2.3.0
+ordered-set==4.1.0
+packaging==23.1
+paginate==0.5.6
+pandas==2.0.3
+panel==1.2.1
+param==1.13.0
+pathspec==0.11.2
+pathtools==0.1.2
+Pillow
+platformdirs==3.10.0
+protobuf==4.23.3
+psutil==5.9.5
+pyarrow==12.0.1
+pyasn1==0.5.0
+pyasn1-modules==0.3.0
+pydantic==2.0.3
+pydantic_core==2.3.0
+pydeck==0.8.0
+Pygments==2.16.1
+PyJWT==2.8.0
+pymdown-extensions==10.3
+Pympler==1.0.1
+pyparsing==3.0.9
+python-dateutil==2.8.2
+python-editor==1.0.4
+python-multipart==0.0.7
+pytorch-lightning==2.0.4
+pytz==2023.3
+pytz-deprecation-shim==0.1.0.post0
+pyviz-comms==2.3.2
+PyYAML==6.0
+pyyaml_env_tag==0.1
+readchar==4.0.5
+redis==4.6.0
+referencing==0.30.0
+regex==2023.8.8
+requests==2.31.0
+requests-oauthlib==1.3.1
+rich==13.4.2
+rpds-py==0.9.2
+rsa==4.9
+s3fs==2023.6.0
+safetensors==0.3.1
+sentry-sdk==1.26.0
+setproctitle==1.3.2
+six==1.16.0
+smmap==5.0.0
+sniffio==1.3.0
+soupsieve==2.4.1
+starlette==0.27.0
+starsessions==1.3.0
+streamlit==1.25.0
+sympy==1.12
+tdqm==0.0.1
+tenacity==8.2.2
+tensorboard==2.13.0
+tensorboard-data-server==0.7.1
+tensorboardX==2.6.2
+tokenizers==0.13.3
+toml==0.10.2
+toolz==0.12.0
+torch==2.0.1
+torchaudio==2.0.2
+torchmetrics==1.0.0
+torchvision==0.15.2
+tornado==6.3.3
+tqdm==4.65.0
+traitlets==5.9.0
+transformer-lens @ git+https://github.com/neelnanda-io/TransformerLens.git@218ebd6f491f47f5e2f64e4c4327548b60a093eb
```

Comment:
Probably shouldn't have transformer lens anymore here right? Let me know if we still need this for some reason